src/components/GradesView/EditModal/OverrideTable/AdjustedGradeInput.jsx

```diff
@@ -20,15 +20,28 @@ export class AdjustedGradeInput extends React.Component {
   }
 
   onChange = ({ target }) => {
-    this.props.setModalState({ adjustedGradeValue: target.value });
+    let adjustedGradeValue;
+    switch (true) {
```

Comment:
why set min/max AND re-enforce onChange?

Reply:
Input attributes min/max validate value which was changed by input number controls but it doesn't validate key entering.
So an additional validation in the `onChange` method was created for validation value from the keyboard.

Reply:
Also, the package-lock.json file was updated because of PR conflicts.